spec/acceptance/enum_traits_spec.rb

```diff
@@ -0,0 +1,161 @@
+describe "enum traits" do
+  context "when automatically_define_enum_traits is true" do
+    it "builds traits automatically for model enum field" do
+      define_model("Task", status: :integer) do
+        enum status: { queued: 0, started: 1, finished: 2 }
+      end
+
+      FactoryBot.define do
+        factory :task
+      end
+
+      Task.statuses.each_key do |trait_name|
+        task = FactoryBot.build(:task, trait_name)
+
+        expect(task.status).to eq(trait_name)
+      end
+
+      Task.reset_column_information
```

Comment:
these may be better in an after hook so it's more readable (obviously not high prio)

Reply:
This made me realize that not all of these tests need models, so I changed some in 1f70160 to use plain old Ruby objects instead. It no longer makes sense to call `Task.reset_column_information` in an after hook since `Task` won't have that method in every test.